modules/ROOT/pages/to-download-an-asset.adoc

```diff
@@ -8,6 +8,8 @@ Download lets you download a copy of the asset, or in the case of a Custom asset
 
 After creating a REST asset, Exchange generates a Mule 3 connector and a Mule 4 connector. Exchange also generates an OAS version for RAML assets, and a RAML version for OAS assets. After you create a new REST asset it may take a few minutes for these assets to appear on the Download menu.
 
+OAS versions were not automatically generated before January 2019. To generate an OAS version of an old asset, download the asset and upload it again.
```

Comment:
@mcampo : When will this happen? If I'm not wrong, this(non-existence of an OAS spec for the corresponding RAML spec) will only occur, if the RAML spec was uploaded before Jan, 2019, and it wasn't ever updated since then. Is that correct? If the user downloads & updates the RAML Spec again, after Jan 2019, it will ONLY generate the OAS spec for that updated version, right? Not any of the older versions of the spec.

Please let me know if the above statement is correct. If so, I'd like to change the text to something like this(feel free to update the English):
After Jan <need to provide an exact date here>, 2019, any API spec(RAML or OAS 2.0) uploaded to Exchange would generate and store the other format of the spec. e.g. if a RAML is uploaded, an OAS will be auto generated, and vice versa. See corresponding release notes here<link to the appropriate release notes for the OAS update>. However, for RAML specs uploaded to Exchange before Jan <date>, 2019, would not have generated a corresponding OAS spec. In order to have an OAS spec auto-generated for that RAML, users will need to download & upload that version, after Jan <date>, 2019. "

Probably a much shorter description.

Reply:
"If I'm not wrong, this(non-existence of an OAS spec for the corresponding RAML spec) will only occur, if the RAML spec was uploaded before Jan, 2019, and it wasn't ever updated since then. Is that correct? If the user downloads & updates the RAML Spec again, after Jan 2019, it will ONLY generate the OAS spec for that updated version, right? Not any of the older versions of the spec."

This is also my understanding.

After Mariano confirms I will update the text to explain the auto generation and link to the release notes as you suggest.

Reply:
Mariano and Lucas Ludueno explained on Slack that OAS was generated from RAML from the beginning on Feb 10, 2017, AMF has been used to generate OAS from RAML since Oct 4, 2018, and RAML has been generated from OAS 2.0 since Jan 12, 2019. Updated the text and added the link to release notes.